### PR TITLE
Tracker: Issues list view

### LINKED ISSUE #1272: Tracker: Issue editor
- [x] Use Activity control(with customizations)
- [x] Issue properties editor
- [x] Description editor
- [x] Title editor
- [ ] Context menu for Dueto/etc.

### PR DESCRIPTION
**Pull request by agrigorovich-xored**

### CODE REVIEW

File: models/tracker/src/index.ts [hunk 1]
```diff
@@ -101,7 +101,7 @@ export class TIssue extends TDoc implements Issue {
   @Prop(Collection(attachment.class.Attachment), tracker.string.Attachments)
   attachments!: number
 
-  // @Prop(Collection(core.class.TypeString), tracker.string.Labels)
+  @Prop(Collection(core.class.TypeString), tracker.string.Labels)
```
Comment (haiodo) [inline models/tracker/src/index.ts]: Better to declare special LabelType with appropriate presenters.

File: plugins/tracker-resources/src/components/issues/AssigneePresenter.svelte [hunk 2]
```diff
@@ -0,0 +1,45 @@
+<!--
+// Copyright © 2020, 2021 Anticrm Platform Contributors.
+// Copyright © 2021 Hardcore Engineering Inc.
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/issues/AssigneePresenter.svelte]: 2022 and only second copyright

File: plugins/tracker-resources/src/components/issues/AssigneePresenter.svelte [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+-->
+<script lang="ts">
+  import { Employee, formatName } from '@anticrm/contact'
+  import { Hierarchy } from '@anticrm/core'
+  import { Avatar } from '@anticrm/presentation'
+  import { showPanel, Tooltip } from '@anticrm/ui'
+  import view from '@anticrm/view'
+  import tracker from '../../plugin'
+
+  export let value: Employee
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/issues/AssigneePresenter.svelte]:
```suggestion
  export let value: WithLookup<Employee>
  // and show value.lookup.assignee
```

File: plugins/tracker-resources/src/components/issues/CategoryPresenter.svelte [hunk 4]
```diff
@@ -1,52 +1,99 @@
...
-</Scroller>
+  <Scroller>
+    <IssuesList
+      _class={tracker.class.Issue}
+      config={[
+        { key: '', presenter: tracker.component.IssuePresenter, props: { currentTeam } },
+        { key: '', presenter: tracker.component.TitlePresenter },
+        { key: '', presenter: tracker.component.LabelsPresenter },
+        { key: 'modifiedOn', presenter: tracker.component.ModificationDatePresenter },
+        '$lookup.assignee'
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/issues/CategoryPresenter.svelte]: ```{ key: '$lookup.assignee', props: { showText: false } }```
Please check view-resources implementation buildModel()

File: plugins/tracker-resources/src/components/issues/ControlCheckBox.svelte [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  const handleChanged = (event: Event) => {
+    const eventTarget = event.target as HTMLInputElement
+    const isChecked = eventTarget.checked
+
+    onSelected?.(id, isChecked)
+  }
+</script>
+
+<div class={controlCheckBoxClass}>
+  <CheckBox checked={isChecked} on:change={handleChanged} />
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/issues/ControlCheckBox.svelte]: Please improve CheckBox to have something like on:value

File: plugins/tracker-resources/src/components/issues/IssuesList.svelte [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+        dispatch('content', docObjects)
+        isLoading = false
+      },
+      { sort, ...options, limit: DOCS_MAX_AMOUNT }
+    )
+  }
+
+  $: updateData(_class, query, options)
+
+  const getValue = (doc: Doc, key: string) => {
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/issues/IssuesList.svelte]: Please use getObjectValue from '@anticrm/core'

File: plugins/tracker-resources/src/components/issues/ModificationDatePresenter.svelte [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import { Tooltip } from '@anticrm/ui'
+  import tracker from '../../plugin'
+
+  export let value: number
+
+  $: shortModificationDate = new Date(value).toLocaleString('default', { month: 'short', day: 'numeric' })
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/issues/ModificationDatePresenter.svelte:21]: Please check existing controls.

Comment (agrigorovich-xored) [reply plugins/tracker-resources/src/components/issues/ModificationDatePresenter.svelte:21]: I couldn't find exactly this format in project.

File: plugins/tracker-resources/src/components/issues/ControlCheckBox.svelte (hunk 5 above)
Comment (agrigorovich-xored) [reply plugins/tracker-resources/src/components/issues/ControlCheckBox.svelte]: I plan to address this issue in the separate PR. Removed `ControlCheckBox` for now.

File: .vscode/settings.json [hunk 8]
```diff
@@ -1,19 +0,0 @@
-{
-  "files.exclude": {
```
Comment (haiodo) [inline .vscode/settings.json:2]: why?

File: plugins/tracker-resources/src/components/issues/CategoryPresenter.svelte [hunk 9]
```diff
@@ -1,52 +1,97 @@
...
-    ]}
-    options={options}
-    query={{ ...query, status: category }}
-    showNotification
-    highlightRows
-    on:content={(evt) => { visible = evt.detail.length > 0 }}
-  />
-</Scroller>
+  <Scroller>
+    <IssuesList
```
Comment (haiodo) [inline plugins/tracker-resources/src/components/issues/CategoryPresenter.svelte:50]: Probable we should rename IssuesList to ListView and put it inside view/view-resources

**Review by haiodo — APPROVED** [review]

**Pull request merged**